src/ZigguratConstants.chpl

```diff
@@ -0,0 +1,721 @@
+module ZigguratConstants {
+    /*
+        Constants / lookup tables for the ziggurat method (https://en.wikipedia.org/wiki/Ziggurat_algorithm)
+        of random number generation sourced from:
+        https://github.com/numpy/numpy/blob/main/numpy/random/src/distributions/ziggurat_constants.h
+
+        It's important that these be tuples so they can be remote value forwarded to other locales
+        and avoid unnecessary communication with locale 0. See PR #2354 for more information
+
+        included constants are:
+            ziggurat_nor_r
+            ziggurat_nor_inv_r
+            ziggurat_exp_r
+            ki_double
+            wi_double
+            fi_double
+            ke_double
+            we_double
+            fe_double
+    */
+
+    const ziggurat_nor_r = 3.6541528853610087963519472518;
+    const ziggurat_nor_inv_r = 0.27366123732975827203338247596; // 1.0 / ziggurat_nor_r;
+    const ziggurat_exp_r = 7.6971174701310497140446280481;
+
+    const ki_double = (
```

Comment:
I wonder if making these large tables `param` values would be beneficial, though, the compiler is likely already doing this since it's just literal values.

Reply:
I went ahead and converted the ones I could. The compiler said the float tuples is not a supported param type